apps/client/src/app/core/services/builds.service.ts

```diff
@@ -0,0 +1,492 @@
+                this.getEngravingIdByName("Cursed Doll")
+            ],
+        },
+        {
+            name: "Reflux Sorceress",
+            stats: [LostArkStat.Crit, LostArkStat.Swiftness],
+            engravings: [
+                this.getEngravingIdByName("Reflux"),
+                this.getEngravingIdByName("Precise Dagger"),
+                this.getEngravingIdByName("Hit Master"),
+                this.getEngravingIdByName("Adrenaline") ,
+                this.getEngravingIdByName("All-Out Attack"),
+                this.getEngravingIdByName("Grudge"),
+                this.getEngravingIdByName("Cursed Doll")
+            ],
+        },
+        // Glaivier
+        {
+            name: "Pinnacle Glaivier",
+            stats: [LostArkStat.Specialiazation, LostArkStat.Swiftness],
+            engravings: [
+                this.getEngravingIdByName("Pinnacle"),
+                this.getEngravingIdByName("Keen Blunt Weapon"),
+                this.getEngravingIdByName("Mass Increase"),
+                this.getEngravingIdByName("Awakening"),
+                this.getEngravingIdByName("Grudge"),
+                this.getEngravingIdByName("Cursed Doll")
+            ]
+        },
+        {
+            name: "Control Glaivier",
+            stats: [LostArkStat.Crit, LostArkStat.Swiftness],
+            engravings: [
+                this.getEngravingIdByName("Control"),
+                this.getEngravingIdByName("Ambush Master"),
+                this.getEngravingIdByName("Adrenaline"),
+                this.getEngravingIdByName("Raid Captain") ,
+                this.getEngravingIdByName("Keen Blunt Weapon"),
+                this.getEngravingIdByName("Grudge"),
+                this.getEngravingIdByName("Cursed Doll")
+            ],
+        },
+        // Destroyer
+        {
+            name: "Hammer of Rage Destroyer",
+            stats: [LostArkStat.Crit, LostArkStat.Swiftness, LostArkStat.Specialiazation],
+            engravings: [
+                this.getEngravingIdByName("Rage Hammer"),
+                this.getEngravingIdByName("Super Charge"),
+                this.getEngravingIdByName("Master Brawler"),
+                this.getEngravingIdByName("Keen Blunt Weapon"),
+                this.getEngravingIdByName("Spirit Absorption"),
+                this.getEngravingIdByName("Grudge"),
+                this.getEngravingIdByName("Cursed Doll")
+            ],
+        },
+        {
+            name: "Gravity Training Destroyer",
+            stats: [LostArkStat.Specialiazation, LostArkStat.Crit],
+            engravings: [
+                this.getEngravingIdByName("Gravity Training"),
+                this.getEngravingIdByName("Barricade"),
+                this.getEngravingIdByName("Master Brawler"),
+                this.getEngravingIdByName("Spirit Absorption"),
+                this.getEngravingIdByName("Grudge"),
+                this.getEngravingIdByName("Cursed Doll")
+            ]
```

Comment:
Missing scouter build. I have no clue regarding that class.

as of https://maxroll.gg/lost-ark/build-guides/evolutionary-legacy-machinist-raid-guide

Evolutionary Legacy Machinist
Stats: spec, crit
Engravings:
* Evolutionary Legacy
* Adrenaline
* Grudge
* Keen Blunt Weapon
* Barricade
* Raid Captain

Arthetinean Skill Machinist
Stats: crit, swiftness
Engravings:
* Arthetinean Skill 
* Adrenaline
* Grudge
* Keen Blunt Weapon
* Raid Captain
* Mass Increase

Reply:
Is missing Arcanist build too. Will add it

Reply:
Done https://github.com/Supamiu/Lostark-helper/pull/52/commits/d918d526e7bddc89886bb94957fd91a96b4a25f2